### DISCUSSION TITLE
What is the intended use of AllergyIntoleranceVerificationStatusCodes and AllergyIntoleranceClinicalStatusCodes?

### DISCUSSION THREAD

**Discussion by Sir-Baconn** [FirelyTeam/firely-net-sdk#2011] [Q&A] [answered]
Using R4

The `VerificationStatus` and `ClinicalStatus` fields are both of type `CodeableConcept`. Thus, the enums `AllergyIntoleranceVerificationStatusCodes` and `AllergyIntoleranceClinicalStatusCodes` can not be used directly.

I see the enum values have an `EnumLiteral` attribute on them which seems to have the desired code value like "entered-in-error" for `AllergyIntoleranceVerificationStatusCodes.EnteredInError`.

Thanks!

**Comment by marcovisserFurore** [comment, marked as answer]
Hi @Sir-Baconn, 

Thanks for your interest in our SDK.

You can use the enum `AllergyIntoleranceVerificationStatusCodes` to populate a CodeableConcept:
```c#
            var enteredInErrorCoding = new Coding(
                AllergyIntoleranceVerificationStatusCodes.EnteredInError.GetSystem(),
                AllergyIntoleranceVerificationStatusCodes.EnteredInError.GetLiteral()
                );

            var allergyIntolerance = new AllergyIntolerance();
            allergyIntolerance.VerificationStatus = new CodeableConcept() { Coding = new List<Coding> { enteredInErrorCoding } };
```

I hope I gave you enough inspiration to continue. 

Marco
